### ISSUE TITLE
Superbuild Error-(error in sb-paraview-build.cmake:47(message))

### ISSUE THREAD

**Issue by alvin198811** [Kitware/VeloView#110]
Hi all

can anybody give me some investigate direction about the build error as bellow.

**Comment by kinger-ml** [comment]
@alvin198811 I am getting the same error. Any updates on this?

**Comment by iVMVi** [comment]
I`ve had the same problem. It started working when I reinstalled Qt to version 5.9.9

**Comment by kinger-ml** [comment]
Yes. It seems to be QT issue. I had the latest version of QT when I got this error. It works fine with Qt 5.10.
